svm/src/account_loader.rs

```diff
@@ -48,6 +48,12 @@ pub struct LoadedTransaction {
     pub rent_debits: RentDebits,
 }
 
+impl LoadedTransaction {
+    pub fn get_fee_payer_account(&self) -> Option<&TransactionAccount> {
```

Comment:
`LoadedTransaction::accounts` is public, why add a getter?

Reply:
Just thought it would be nice to have an explicit self-documenting method for it

Reply:
Then I would say we move it to the definition site of `LoadedTransaction` and rename `get_fee_payer_account()` to `fee_payer_account()`.